src/plugins/data_views/public/services/has_data.ts

```diff
@@ -156,22 +158,33 @@ export class HasData {
   private getHasDataViews = async ({ http }: { http: HttpStart }): Promise<HasDataViewsResponse> =>
     http.get<HasDataViewsResponse>(`/internal/data_views/has_data_views`);
 
-  private findDataViews = (http: HttpStart): Promise<boolean> => {
+  private hasDataViews = (http: HttpStart): Promise<boolean> => {
     return this.getHasDataViews({ http })
       .then((response: HasDataViewsResponse) => {
         const { hasDataView } = response;
         return hasDataView;
       })
-      .catch(() => false);
+      .catch((e) => {
+        // eslint-disable-next-line no-console
+        console.warn(`hasDataViews failed with error, assuming there are data views`, e);
+        return true;
```

Comment:
🙌

Reply:
Also fyi, most of the UI is already catching this in the same way.

Reply:
It is more often `false` in consuming code. those catches are redundant, but if we'd remove them from services, then probably should indeed change most of those to `true` 

<img width="958" alt="Screenshot 2022-08-02 at 16 02 32" src="https://user-images.githubusercontent.com/7784120/182393829-3340e479-7b1f-4c22-8b51-658044ce4a04.png">